src/libcollections/fmt.rs

```diff
@@ -423,8 +418,10 @@
 //!
 //!    `.*` means that this `{...}` is associated with *two* format inputs rather than one: the
 //!    first input holds the `usize` precision, and the second holds the value to print.  Note that
-//!    in this case, if one uses the format string `{<arg>:<spec>.*}`, then the `<arg>` part refers
-//!    to the *value* to print, and the `precision` must come in the input preceding `<arg>`.
+//!    in this case, if one uses the format string `{<arg>:<format_spec>.*}`, then the `<arg>` part
+//!    refers to the *value* to print. If `<arg>` is a named parameter the `precision` must come in
+//!    the input preceding `<arg>`. It is not possible to specify a positional or named parameter
```

Comment:
> It is not possible to specify a positional or named parameter for the precision.

Also note [the "positional parameters" section](https://doc.rust-lang.org/std/fmt/index.html#positional-parameters) of `std::fmt` seems to categorize `{}` as a positional parameter. So what `.*` does is, to my eyes, nothing but specifying a positional parameter as the precision.